inji-web/package.json

```diff
@@ -33,7 +33,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "set PORT=3001 && react-scripts start",
+    "start": "set PORT=3001 && export PORT=3001 && react-scripts start",
```

Comment:
export might not be available in other OS. this might fail to run the NPM command. Can we check?

Reply:
corrected it.